examples/typescript/example.ts

```diff
@@ -0,0 +1,27 @@
+import { Client } from "pg";
+
+async function main() {
+  const localHostStr =
+    "postgresql://user:pass@my_org.proxy.glaredb.com:6543/my_db"; // Replace with your connection string
```

Comment:
Are we able to make this runnable like the Go one?

Reply:
Done here: https://github.com/GlareDB/glaredb/pull/1461